packages/desmos-std/src/mock.rs

```diff
@@ -1,163 +1,195 @@
-                attachments: Some(vec![]),
-                poll: Some(Poll {
-                    question: "".to_string(),
-                    provided_answers: vec![],
-                    end_date: "".to_string(),
-                    allows_multiple_answers: false,
-                    allows_answer_edits: false,
-                }),
-                creator: "default_creator".to_string(),
-            };
-            to_binary(&PostsResponse { posts: vec![post] })
-        }
-        DesmosQuery::Reports { post_id } => {
-            let report = Report {
-                post_id,
-                kind: "test".to_string(),
-                message: "test".to_string(),
-                user: "default_creator".to_string(),
-            };
-            to_binary(&ReportsResponse {
-                reports: vec![report],
-            })
-        }
-        DesmosQuery::Reactions { post_id } => {
-            let reactions = vec![Reaction {
-                post_id,
-                short_code: ":heart:".to_string(),
-                value: "❤️".to_string(),
-                owner: "user".to_string(),
-            }];
-            to_binary(&ReactionsResponse { reactions })
-        }
-    };
-    response.into()
-}
-
 #[cfg(test)]
 mod tests {
-    use super::*;
-    use crate::query_types::{PostsResponse, ReportsResponse};
-    use crate::types::{DesmosRoute, Report};
-    use cosmwasm_std::{from_binary, QuerierWrapper};
+    use crate::profiles::models_query::QueryUserApplicationLinkResponse;
+    use crate::{
+        mock::mock_dependencies_with_custom_querier,
+        profiles::{
+            mock::MockProfilesQueries,
+            models_query::{
+                QueryApplicationLinkByClientIDResponse, QueryApplicationLinksResponse,
+                QueryBlocksResponse, QueryChainLinksResponse,
+                QueryIncomingDtagTransferRequestResponse, QueryProfileResponse,
+                QueryRelationshipsResponse, QueryUserChainLinkResponse,
+            },
+            querier::ProfilesQuerier,
+        },
+    };
+    use cosmwasm_std::Addr;
+    use std::ops::Deref;
 
     #[test]
-    fn custom_query_execute_posts() {
-        let post = Post {
-            post_id: "id123".to_string(),
-            parent_id: Some("id345".to_string()),
-            message: "message".to_string(),
-            created: "date-time".to_string(),
-            last_edited: "date-time".to_string(),
-            comments_state: "ALLOWED".to_string(),
-            subspace: "subspace".to_string(),
-            additional_attributes: Some(vec![]),
-            attachments: Some(vec![]),
-            poll: Some(Poll {
-                question: "".to_string(),
-                provided_answers: vec![],
-                end_date: "".to_string(),
-                allows_multiple_answers: false,
-                allows_answer_edits: false,
-            }),
-            creator: String::from("default_creator"),
+    fn test_query_profile() {
```

Comment:
In the general `mock` lib, the thing only should be tested is if the binding of `mock dependency` to the module mock querier works properly or not.
We can remain only one test function to test the binding, then move the rest of them to the module `mock lib`.

Reply:
@dadamu Thanks for pointing this out. I'm moving the querier tests inside `querier.rs` and add some proper tests here